### PR TITLE
Extend, not override LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by gabrielhartmann**
Additionally, simplify generic prefixes on Scheduler cmds

### CODE REVIEW

**Comment by gabrielhartmann** [comment]
@mohitsoni:  We should remove the hack in Gilbert's dse implementation regarding LD_LIBRARY_PATH once this is merged.

**Comment by gabrielhartmann** [comment]
@gmflau: You will want to rebase on master after this PR goes in.

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java [hunk 1]
```diff
@@ -470,7 +470,7 @@ private static boolean isPortResource(Protos.Resource resource) {
         // command and user:
 
         Protos.CommandInfo.Builder commandInfoBuilder = executorInfoBuilder.getCommandBuilder()
-                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib && " +
+                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib:$LD_LIBRARY_PATH && " +
```
Comment (mrbrowning) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java]: Not for this PR, but should we consider making the executor command prefix configurable with a reasonable default? Most people won't need or want to touch it, but those that do would probably appreciate not being forced into using our bundle.

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java (hunk 1 above)
Comment (gabrielhartmann) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java]: @mrbrowning: It's definitely a thought worth discussion.  This changes something pretty fundamental to the operation of a Framework.  We try to keep each version of a package very well defined.  As long as a change here were statically built into a package I'd be happy, but I worry about N customers defining N different ways to start the executor.  Could be a support problem

**Comment by mohitsoni** [comment]
I going to close https://github.com/mesosphere/dcos-commons/pull/465 as this PR fixes the same problems.

File: frameworks/hdfs/universe/config.json [hunk 2]
```diff
@@ -14,6 +14,11 @@
           "description":"The principal for the HDFS service instance.",
           "type":"string",
           "default":"hdfs-principal"
+        },
+        "cmd_prefix" : {
```
Comment (nickbp) [inline frameworks/hdfs/universe/config.json]: Feels a little iffy putting this option front and center in the `service` section but there currently isn't an obviously better place to put it. If more options like this are added we may want to break them out into an `advanced` section at some point but that's more of a UI/product thing.

Comment (gabrielhartmann) [inline frameworks/hdfs/universe/config.json]: Yeah, I don't want this here, but as you say, I have nowhere else to put it.  I think it's better than in Marathon.json.mustache which was getting to be a huge ugly command.  Also the Template framework will just get this, things will stay a bit neater.

**Pull request merged**